### ISSUE TITLE
Exception when documenting under Windows

### ISSUE THREAD

**Issue by bernhard-hofmann** [jacobrask/styledocco#9]
I'm trying to use styledocco on Windows with Node v0.6.11. I created a simple file `default.css`:

```
body {
   font-family:Arial,Sans-serif;
}
```

Then I ran styledocco:

```
C:\temp\node>styledocco -n FirstTest default.css

node.js:201
        throw e; // process.nextTick error, or 'error' event on first tick
              ^
Error: ENOENT, no such file or directory 'C:\temp\node\undefined'
    at Object.openSync (fs.js:230:18)
    at Object.readFileSync (fs.js:120:15)
    at C:\Documents and Settings\bernhard.hofmann\Application Data\npm\node_modules\styledocco\lib\main.js:28:15
    at Object.<anonymous> (C:\Documents and Settings\bernhard.hofmann\Application Data\npm\node_modules\styledocco\lib\main.js:120:14)
    at Object.<anonymous> (C:\Documents and Settings\bernhard.hofmann\Application Data\npm\node_modules\styledocco\lib\main.js:146:4)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Module.require (module.js:354:17)
```

I'm a complete Node noob, so maybe I'm doing something wrong where. I am able to run JS files through Node, so I suspect my Node installation is sound.

The file it's trying to read (`C:\temp\node\undefined`) does not exist of course, but why it's trying to read an undefined file is what I don't understand.

**Comment by jacobrask** [comment]
Hi,

I think it might have been related to a bug that I fixed in f6d1c82065. It should be fixed in 0.2.2 that I just released.

Would you mind trying again and reporting back here?

Thanks!

**Comment by bernhard-hofmann** [comment]
You da MAN!
